server/etcdserver/apply/apply.go

```diff
@@ -207,6 +208,11 @@ func (a *applierV3backend) LeaseRevoke(lc *pb.LeaseRevokeRequest) (*pb.LeaseRevo
 	return &pb.LeaseRevokeResponse{Header: a.newHeader()}, err
 }
 
+func (a *applierV3backend) LeaseRenew(lc *pb.LeaseKeepAliveRequest) (*pb.LeaseKeepAliveResponse, error) {
```

Comment:
I think it's still possible to apply `LeaseRenew` entries issued by a stale leader unless it doesn't have a mechanism like comparing term https://github.com/etcd-io/etcd/issues/15247#issuecomment-1510362310 ?
I think all Raft messages issued by etcd itself might have similar problems potentially.